mkdocs.yml

```diff
@@ -23,6 +23,15 @@ theme:
     primary: indigo
     accent: deep orange
 
+#Enable macros
+plugins:
+  - macros    
```

Comment:
You need to explicitly add the search plugin again according to the [documenation](https://www.mkdocs.org/user-guide/configuration/#plugins).